static/bidder-info/lm_kiviads.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://pbs.kiviads.live/?pid={{.SourceId}}&host={{.Host}}"
+maintainer:
+  email: "prebid@kiviads.com"
```

Comment:
Prebid team has sent verification mail on above email address.  Respond back on email with "received" message to complete verification

Reply:
done